browser/src/react-redux/App/Signup/Form.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Form = props => {
+  return (
+    <div className="form">
```

Comment:
there may be another `form` class if we stay this generic, that will conflict. Let's be more specific. How about `signup-form`?

Reply:
Just to emphasis above, this will be re-named something else as this isn't an actual form.